src/explore-education-statistics-admin/src/pages/publication/components/PublicationDetailsForm.tsx

```diff
@@ -157,21 +159,18 @@ export default function PublicationDetailsForm({
                 </ButtonGroup>
               </RHFForm>
 
-              <ModalConfirm
-                title="Confirm publication changes"
-                onConfirm={async () => {
-                  await form.handleSubmit(handleSubmit)();
-                }}
-                onExit={toggleConfirmModal.off}
-                onCancel={toggleConfirmModal.off}
-                open={showConfirmModal}
-              >
-                <p>
-                  Any changes made here will appear on the public site
-                  immediately.
-                </p>
-                <p>Are you sure you want to save the changes?</p>
-              </ModalConfirm>
+              {showConfirmModal && (
+                <PublicationUpdateConfirmModal
+                  title={initialValues.title}
```

Comment:
At first glance I thought this was the title for the Modal rather than for publication, It might be clearer if it was something like `initialPublicationTitle` and change the new one to `newPublicationTitle`.